### PR TITLE
Fully enable list type everywhere

### PR DESCRIPTION
**Pull request by herndlm**

### CODE REVIEW

File: phpstan.neon.dist [hunk 1]
```diff
@@ -9,4 +9,4 @@ parameters:
     level: 8
     featureToggles:
         alwaysTrueAlwaysReported: true
-        instanceofType: true
```
Comment (szepeviktor) [inline phpstan.neon.dist:12]: 👀

**Comment by szepeviktor** [comment]
All green 🍏 

Thank you Martin.

File: phpstan.neon.dist (hunk 1 above)
Comment (herndlm) [reply phpstan.neon.dist:12]: yeah, sorry. yesterday was a bad day :D

Comment (herndlm) [reply phpstan.neon.dist:12]: I actually didn't enable the list type in that other PR

**Pull request merged**